custom/icds_reports/static/icds_reports/mobile/css/map-chart.less

```diff
@@ -77,16 +77,22 @@ a:active, a:hover, a:focus {
     height: 25px;
 }
 
-// maps popup handling. modified from datamaps css.
-.locPopup {
+// maps and charts popup handling. modified from datamaps css.
+.locPopup, .chartPopup {
   position: absolute;
   z-index: @zindexMapsPopup;
+  pointer-events: inherit !important;  // nvd3 disables pointer-events on some shared classes
```

Comment:
it took me hours to figure this out. had no idea it was a thing and thrashed around trying to figure out why angular click events weren't binding properly...